frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/ticketView/ticketView.component.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Viewer as ViewerService } from '@/v4/services/viewer/viewer';
+import EditViewpointIcon from '@assets/icons/outlined/rotate_arrow-outlined.svg';
+import CreateViewpointIcon from '@assets/icons/outlined/add_circle-outlined.svg';
+import GotoViewpointIcon from '@assets/icons/outlined/aim-outlined.svg';
+
+import { stripBase64Prefix } from '@controls/fileUploader/imageFile.helper';
+import { MenuItem } from '@mui/material';
+import { useEffect } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { isEmpty, isEqual } from 'lodash';
+import { getImgSrc } from '@/v5/store/tickets/tickets.helpers';
+import { ActionMenuItem } from '@controls/actionMenu';
+import { Viewpoint } from '@/v5/store/tickets/tickets.types';
+import { BasicTicketImage } from '../basicTicketImage/basicTicketImage.component';
+import { TicketImageAction } from '../basicTicketImage/ticketImageAction/ticketImageAction.component';
+import { ActionMenu } from '../basicTicketImage/ticketImageAction/ticketImageAction.styles';
+import { TicketImageActionMenu } from '../basicTicketImage/ticketImageActionMenu.component';
+
+type ITicketView = {
+	value: Viewpoint | undefined;
+	label: string;
+	error: boolean;
+	helperText: string;
+	required: boolean;
+	onBlur: () => void;
+	onChange: (newValue) => void;
+};
+
+export const TicketView = ({
+	value,
+	error,
+	helperText,
+	required,
+	onBlur,
+	onChange,
+	...props
+}: ITicketView) => {
+	const updateViewpoint = async () => {
+		const currentViewpoint = await ViewerService.getViewpoint();
+		if (isEqual(currentViewpoint, value)) return;
```

Comment:
value can contain a screenshot property whereas currentViewpoint does not. This means if you open a ticket that already has a viewpoint set and instantly try to update the viewpoint (without moving the camera position) you get this error
![image](https://user-images.githubusercontent.com/62481250/204266650-30d3dbf5-37dd-4902-bb09-841e24aa9f98.png)
because this condition passes true

Reply:
I believe this also removes the 'screenshot' from the value which is what causes the image to flicker when updating the viewpoint. If you log `value` in this component you can see that screenshot mysteriously disappears and reappears after the component rerenders a few times

Reply:
@sanmont3drepo When I do this now (set a new viewpoint that has the exact same camera) i get the error `properties.Default View.camera.position is a required field`